src/main/resources/changelog.sql

```diff
@@ -0,0 +1,27 @@
+--liquibase formatted sql
```

Comment:
Remove this file from the build.  It gets included in the built extension jar file.